cli/cmd/generate.go

```diff
@@ -63,21 +63,42 @@ func runGenerate(cmd *cobra.Command, args []string) error {
 	}
 }
 
-func genSource(cmd *cobra.Command, path string, pm *plugins.PluginManager, registry specs.Registry, outputFile string) error {
-	if registry == specs.RegistryGithub && !strings.Contains(path, "/") {
-		path = "cloudquery/" + path
-	}
-	version := "latest"
-	if strings.Contains(path, "@") {
-		version = strings.Split(path, "@")[1]
-	}
-
-	sourceSpec := specs.Source{
+func getSourceSpec(path string, registry specs.Registry) specs.Source {
```

Comment:
This needs a test, I'll do it in a follow up PR as this blocks testing pre-releases